### ISSUE TITLE
PackageNotFoundError similar output

### ISSUE THREAD

**Issue by tyler-thetyrant** [conda/conda#5391]
Shouldn't

```
conda install gawk
Fetching package metadata ...........

PackageNotFoundError: Package missing in current osx-64 channels:
  - gawk
```

and
```
conda install -c conda-forge gawk
Fetching package metadata .............

PackageNotFoundError: Package missing in current osx-64 channels:
  - gawk

```
return different outputs? 

I am puzzled by the same (i.e. current osx-64 channels) output.

I think this can be easily fixed [here](https://github.com/conda/conda/blob/a03377a02d7eeb7ef7c2036fc612d04fd7ea7b50/conda/exceptions.py#L404).

**Comment by kalefranz** [comment]
Actually, I think the message should be the same, but it shouldn't have the subdir `osx-64` part.  It should just say missing in current channels.  I guess we can pad with more relevant information.  But I'm not completely sure what that would be.

**Comment by msarahan** [comment]
Please do not remove subdir.  That is really important information when conda-build is doing things for the non-native platform.  You should instead add more information to this dialog to say exactly what channels are in the collection.

**Comment by tyler-thetyrant** [comment]
This was addressed in conda 4.3.x. This will be merged into conda 4.4.x soon.
